src/index.js

```diff
@@ -1,19 +1,28 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import reportWebVitals from './reportWebVitals';
+import { createStore, applyMiddleware } from 'redux';
+import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
 
-import App from './Redux_Todos/App';
-import "./Redux_Todos/index.css"
+import App from './Redux_Todos_Thunk/App';
+import "./Redux_Todos_Thunk/index.css"
+import todoReducer from './Redux_Todos_Thunk/reducers/todoReducer';
```

Comment:
Something wrong with your paths, they are should be shorter

`./index.css`

`./todoReducer`